server/lib/field_hub_web/live/project_show_live.html.heex

```diff
@@ -54,11 +54,49 @@
-                                file_size: db_file_size
+                                file_size: db_file_size,
+                                last_n_changes: last_n_changes
                             },
                             files: files
                         } -> %>
-                            <section>Database documents: <%= db_doc_count %></section>
+                            <section>Database documents: <%= db_doc_count %></section>                      
+                            <section>
+                                
+                                <tr>
+                                    <td valign="top" class="column-last-changes-slider">
+                                        <form id="last_changes_form" phx-change="last_changes">                                        
+                                            <%= @nb_changes_to_display %> last changes                                  
+                                        
+                                            <div class="slider-container">                                             
+                                             <input 
+                                                type="range" 
+                                                min="2" 
+                                                max= "50"
+                                                value= {@nb_changes_to_display}
+                                                class="slider"
+                                                name="n-last-changes"
+                                            > 
+                                            </div>                                       
+                                        </form>
+                                    </td>
+                                    <td> 
+                                        <table>
+                                            <%= for change <- last_n_changes do %> 
+                                                <%{type, date_time, user_name} = CouchService.extract_most_recent_change_info(change) %>
+                                                <tr> 
+                                                    <td>
+                                                        '<%= change["doc"]["resource"]["identifier"] %>' 
+                                                        <br />
+                                                        <%= type %> by user: <i><%= user_name %></i>
+                                                    </td>
+                                                    <td class="column-date-time"><%= Calendar.strftime(date_time, "%c") %></td>                    
+                                                </tr>
+                                            <% end %>                                            
+                                        </table>
+                                    </td>
```

Comment:
We should evaluate the database every 10 seconds as is done already. And the slider only changes how many evaluated changes are displayed. 

For this, we should call Project.evaluate/2 with a maximum `n` of 50 (or whatever the highest slider value).

Reply:
I would like to change the slider with a pagination for the table and a drop down menu for displaying a fixed number of line (e.g  5, 25, 100).